README.md

```diff
@@ -67,8 +67,7 @@ Spin up containers and the app, and then set up the application and solid-* data
 
 ```shell
 docker compose up -d
-bin/rails db:prepare db:seed spec:prepare
-bin/dev
+bin/setup
```

Comment:
Recommend using `bin/setup` over `bin/dev` since it automates playwright setup and other steps.